### PR TITLE
Table cell view post-fixer will not crash if an element inside a cell got attribute and was removed at the same time

### LINKED ISSUE #191: Table cell view post-fixer crashes when element was changed and removed
Related to https://github.com/ckeditor/ckeditor5-table/issues/187.

In short, when a view element which is inside a table gets or has an attribute removed (or the model element was renamed), the view post-fixer will want to perform some action on view element bound to it.

However, if the model element was also removed, the model and view elements will not be mapped anymore, which causes a crash in the post-fixer.

Post-fixer should not fire for view elements which are removed.

### PR DESCRIPTION
**Pull request by scofalik**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fixed: Table cell view post-fixer will not crash if an element inside a cell got attribute and was removed at the same time. Closes #191.

### CODE REVIEW

File: src/converters/tablecell-post-fixer.js [hunk 1]
```diff
@@ -126,6 +126,10 @@ function getElementsToCheck( view ) {
 // - span: for single paragraph with no attributes.
 // - p   : in other cases.
 function ensureProperElementName( currentViewElement, mapper, writer ) {
+	if ( !currentViewElement.root.is( 'rootElement' ) ) {
```
Comment (pjasiun) [inline src/converters/tablecell-post-fixer.js]: Could you leave here a comment about how root can not be `rootElement` and why `false` should be returned in such case?

Comment (scofalik) [inline src/converters/tablecell-post-fixer.js]:
```suggestion
	// This situation may happen if a view element was changed and removed at the same time. In this case the view element is already unbound so the post-fixer would crash.
	if ( !currentViewElement.root.is( 'rootElement' ) ) {
```

Comment (scofalik) [reply src/converters/tablecell-post-fixer.js]: Done.

File: src/converters/tablecell-post-fixer.js [hunk 2]
```diff
@@ -126,6 +126,11 @@ function getElementsToCheck( view ) {
 // - span: for single paragraph with no attributes.
 // - p   : in other cases.
 function ensureProperElementName( currentViewElement, mapper, writer ) {
+	// This situation may happen if a view element was changed and removed at the same time. In this case the view element is already unbound so the post-fixer would crash.
```
Comment (scofalik) [inline src/converters/tablecell-post-fixer.js]:
```suggestion
	// This situation may happen if a view element was changed and removed at the same time.
	// In this case, the view element is already unbound so the post-fixer would crash.
```

**Pull request merged**